src/test/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/ResumeServiceImplTests.java

```diff
@@ -29,19 +35,25 @@
     private ApiClientService apiClientService;
 
     @Mock
-    private ApiClient apiClient;
+    private CompanyService companyService;
 
     @Mock
-    private CompanyResourceHandler companyResourceHandler;
+    private ApiResponse<CompanyProfileApi> responseWithData;
 
     @Mock
-    private CompanyGet companyGet;
+    private CompanyProfileApi companyProfileApi;
 
     @Mock
-    private ApiResponse<CompanyProfileApi> responseWithData;
+    private DirectorsReportService directorsReportService;
 
     @Mock
-    private CompanyProfileApi companyProfileApi;
+    private ProfitAndLossService profitAndLossService;
+
+    @Mock
+    private DirectorsReportApi directorsReportApi;
+
+    @Mock
+    private ProfitAndLoss profitAndLoss;
 
     @InjectMocks
     private ResumeServiceImpl resumeService = new ResumeServiceImpl();
```

Comment:
I don't think you need to create the Object @InjectMocks should do that for you ?